### DISCUSSION TITLE
Spanish language

### DISCUSSION THREAD

**Discussion by crishnakh** [benjaminjonard/koillection#470] [Ideas]
Hi, I recently found and installed Koillection and think about made the spanish translation. It is one of the most used languages and it should be good to share to target more people. I made a fork to translate de files to spanish: https://github.com/Iguannaweb/koillection_es 

I'm translating files of the folder: https://github.com/Iguannaweb/koillection_es/tree/main/translations but do I need to translate anything more?

**Comment by benjaminjonard** [comment]
Hello, 

Thank you very much for your work and time. I'll have a look next week and add the option the choose Spanish (I'm on vacation right now)

Yes, everything that needs to be translated should be located in the translations folder

**Reply by crishnakh** [reply]
Hi @benjaminjonard, that sounds great!
By the time you come back from your vacations I think all will be translated!

**Comment by crishnakh** [comment]
If you add new strings, just tell me to translate it

**Comment by crishnakh** [comment]
Is there any option in crowd in to notify me when new strings needs translations?

**Reply by benjaminjonard** [reply]
As a translator I don't know if you can subscribe to notifications. 
I can activate on my side for everyone, but I don't know if it's a good idea.

![Screenshot_20230815-121424~2](https://github.com/benjaminjonard/koillection/assets/20560781/3c948af8-2698-45d4-a581-32136926ed63)

**Comment by crishnakh** [comment]
mmmm don't know if it's better make a discussion thread to notify them. could be better option? I think crowd in give us options to unsuscribe from notifications if other translator want.

**Comment by benjaminjonard** [comment]
Ok, I enabled notifications when new strings are added, will see how it goes

**Reply by crishnakh** [reply]
Great! I receive it!
